client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -54,18 +54,18 @@ export default function getThankYouPageUrl( {
 }: {
 	siteSlug: string | undefined;
 	adminUrl: string | undefined;
-	redirectTo: string | undefined;
+	redirectTo?: string | undefined;
```

Comment:
Can we get away with `redirectTo?: string` here?

Reply:
Maybe... but they mean different things. If it was `redirectTo?: string` then `const redirectTo = undefined; getThankYouPageUrl({ redirectTo })` would not be allowed, and that could happen here since `redirectTo` is passed down from the router.